Description: Fixed automake warnings, enabled AM_PROG_AR.
Author: Eugene Seliverstov <theirix@gmail.com>
Last-Update: 2013-06-17
Forwarded: https://github.com/vlm/asn1c/pull/109
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/configure.ac
+++ b/configure.ac
@@ -5,8 +5,6 @@
 AM_INIT_AUTOMAKE([foreign serial-tests])
 AC_CONFIG_MACRO_DIR([m4])
 
-LT_INIT
-
 dnl Checks for programs.
 AC_PROG_CC
 AC_PROG_CPP
@@ -15,7 +13,11 @@
 AC_PROG_MAKE_SET
 AC_PROG_YACC
 AM_PROG_LEX
-AC_PATH_PROG(AR, ar, ar, $PATH:/usr/ucb:/usr/ccs/bin)	dnl for Solaris
+m4_ifdef([AM_PROG_AR],
+				 [AM_PROG_AR],
+				 [AC_PATH_PROG(AR, ar, ar, $PATH:/usr/ucb:/usr/ccs/bin)])
+
+LT_INIT
 
 dnl If you need to see the details, just run make V=1.
 m4_ifdef([AM_SILENT_RULES], [AM_SILENT_RULES([yes])])
